src/Map.tsx

```diff
@@ -26,7 +28,12 @@ const Map: React.FC<React.HTMLProps<HTMLDivElement> & MapProps> = ({
 
   useEffect(() => {
     if (mapRef.current !== null) {
-      setMapInstance(new L.Map(mapRef.current, options))
+      const map = new L.Map(mapRef.current, options)
```

Comment:
I think this should be done conditionally if `setInstance` is not passed.

Reply:
I wouldn't do that , the Map component is responsible for instancing the L.Map. In some cases a pan or other state action must be triggered from outside the map.

Reply:
Ah, yes. I see it's just passing the map instance along. That's fine.